src/cljs/athens/devcards/sci_boxes.cljs

```diff
@@ -0,0 +1,310 @@
+(ns athens.devcards.sci-boxes
+  (:require
+    [cljsjs.react]
+    [cljsjs.react.dom]
```

Comment:
I'm not sure why these react imports are necessary but I get "React is not defined" when they're not there